src/main/java/edu/kis/vh/nursery/defaultCountingOutRhymer.java

```diff
@@ -2,32 +2,36 @@
 
 public class defaultCountingOutRhymer {
 
-	private int[] NUMBERS = new int[12];
+	static final int NUMBERS_AMOUNT = 12;
+	static final int CALL_UNSUCCESSFUL = -1;
+	static final int BUFFER_FULL = 11;
 
-	public int total = -1;
+	private int[] NUMBERS = new int[NUMBERS_AMOUNT];
+
+	private int total = CALL_UNSUCCESSFUL;
```

Comment:
this initial value should have its own constant, named like INITIAL_VALUE or something similar